codec-fix/README.md

```diff
@@ -61,6 +61,9 @@ Determines if nanoseconds should be added to date-time / time fields during enco
 
 ## Release notes
 
++ 3.10.0
+  + Fixed problem with truncation of fractions of seconds in time fields
```

Comment:
```suggestion
  + Fixed problem with truncation of fractions of seconds in time fields
```
  + Fixes the truncated time field problem related to incorrect work of the `includeMilliseconds` option